README.md

```diff
@@ -31,13 +34,16 @@ out:
   apikey: 'XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX-XXX'
   list_id: 'XXXXXXXXXX'
   update_existing: false
+  double_optin: false
+  email_column: e mail
+  fname_column: first name
+  lname_column: last name
   merge_fields:
-  - FNAME
-  - LNAME
-  - WEB
+  - website
```

Comment:
It should be lower case. The plugin will be convert to upper case before pushing data